tests/functional/test_install_wheel.py

```diff
@@ -511,9 +495,9 @@ def test_install_from_wheel_uninstalls_old_version(script, data):
     package = data.packages.joinpath("simplewheel-2.0-py2.py3-none-any.whl")
     result = script.pip('install', package, '--no-index')
     dist_info_folder = script.site_packages / 'simplewheel-2.0.dist-info'
-    assert dist_info_folder in result.files_created
+    result.did_create(dist_info_folder)
     dist_info_folder = script.site_packages / 'simplewheel-1.0.dist-info'
```

Comment:
We should have a follow up PR that changes the variable names in this test, to clearly indicate what is "setup" and what's checking the behavior here, and also check that things get deleted.

Reply:
Agreed. I'll create a PR once this gets merged. :)